src/App.tsx

```diff
@@ -1,24 +1,46 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
 import React from 'react';
-import { UserWarning } from './UserWarning';
+import cn from 'classnames';
+import { useTodoContext } from './context/TodoContext';
 
-const USER_ID = 0;
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { NotificationModal } from './components/Notification/Notification';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const { todos, nrOfActiveTodos, toggleAll } = useTodoContext();
 
   return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-loading-todos#react-todo-app-load-todos">React Todo App - Load Todos</a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+        <header className="todoapp__header">
+          {(todos.length > 0) && (
+            <button
+              type="button"
+              className={
+                cn('todoapp__toggle-all',
+                  { active: nrOfActiveTodos === todos.length })
+              }
+              aria-label="Toggle between active and not active"
+              onClick={toggleAll}
+            />
+          )}
+
+          <TodoForm />
+        </header>
+
+        <section className="todoapp__main">
+          <TodoList />
+        </section>
+
+        {todos.length > 0 && (
```

Comment:
`todos.length` will be enough for this check.

Reply:
need to use !! operator. Otherwise prints "0"

Reply:
Valid point! Got so used with conditional statements that overlooked the default behaviour here. 

If interested, check this package: https://www.npmjs.com/package/jsx-control-statements
I really encourage using it other that just having conditions on the jsx.